interface/patient_file/summary/pnotes.php

```diff
@@ -173,6 +175,36 @@
 
 </div> <!-- end pnotes -->
 
+
+<script type="text/javascript">
+document.addEventListener('DOMContentLoaded', function() {
+    console.log("DOM fully loaded and parsed");
+
+    var knownDomains = ['open-emr.org'];
```

Comment:
Yea, we can't take this PR as is.  Because of the self-hosting nature, this needs to come from a list of accepted domains in OpenEMR and needs to be a feature that can be turned on / off due to its brunt nature of attaching to ALL anchor tags whether the anchor is a hyperlink or a button

Reply:
@sravyakaranam let us know if this is something you'd want to investigate & fix, otherwise we'll close the PR in a few weeks.